visualization/app/codeCharta/ui/codeMap/rendering/codeMapMesh.ts

```diff
@@ -121,15 +121,20 @@ export class CodeMapMesh {
 	}
 
 	private setNewDeltaColor(building: CodeMapBuilding, state: State) {
-		if (building.node.deltas) {
-			const deltaValue = building.node.deltas[state.dynamicSettings.heightMetric]
+		const { appSettings : {mapColors}, dynamicSettings : {heightMetric} } = state;
+		const { node } = building;
+		
+		if (node.flat)
```

Comment:
I think, we should not omit the brackets. What do you think?

Reply:
This should likely be fixed by running prettier. Seems like the pre-commit hook was not executed (we still have that, right?).

Reply:
Yes we do, however theneed to be enabled if you dont commit via commandline